iceoryx_hoofs/test/moduletests/test_objectpool.cpp

```diff
@@ -17,9 +17,7 @@
 
 #if !defined(_WIN32)
```

Comment:
Could you remove this and check if it now runs on Windows? AFAIK the `#define private public` was the only reason why it was disabled for Windows

Reply:
done